neqo-transport/src/streams.rs

```diff
@@ -61,6 +61,10 @@ impl Streams {
         }
     }
 
+    pub fn is_in_remote_bidi_limit(&self, stream_id: StreamId) -> bool {
+        self.remote_stream_limits[StreamType::BiDi].is_in_limit(stream_id)
```

Comment:
maybe call the function `is_stream_id_allowed`
You are not checking the type of the stream in this function. The caller is responsible for that, that is fine, but you can make it generic:
```
pub fn is_stream_id_allowed(&self, stream_id: StreamId) -> bool {
    self.remote_stream_limits[stream_id.stream_type()].is_in_limit(stream_id)
}
```